compose/ui/ui-test-junit4/build.gradle

```diff
@@ -78,6 +78,10 @@ if(AndroidXComposePlugin.isMultiplatformEnabled(project)) {
     androidXComposeMultiplatform {
         android()
         desktop()
+        darwin()
```

Comment:
When I run `desktop/test` from IDEA configuration, I have this error:
```
compose-jb\compose\frameworks\support\compose\ui\ui-test-junit4\src\skikoMain\kotlin\androidx\compose\ui\test\ComposeUiTest.skikoMain.kt: (41, 25): Unresolved reference: Clock
```
![image](https://user-images.githubusercontent.com/5963351/203595943-389f5370-7f1f-40f5-bd8a-00cf704b5d71.png)

Reply:
hm.. no issue in my IDEA

the line is: `import kotlinx.datetime.Clock` 

and in build.gradle in skikoMain we have: `implementation("org.jetbrains.kotlinx:kotlinx-datetime:0.4.0")`

Reply:
Does it keep failing for you?

Reply:
I got rid of `kotlinx-datetime` dependency. We have `currentNanoTime()` function in skiko, so we can rely on it instead.

Reply:
Yes, now it works )